src/whisper.cpp

```diff
@@ -1309,14 +1309,19 @@ static size_t aheads_masks_nbytes(struct whisper_aheads_masks & aheads_masks) {
 static ggml_backend_t whisper_backend_init_gpu(const whisper_context_params & params) {
     ggml_log_set(g_state.log_callback, g_state.log_callback_user_data);
 
+    #ifndef BINDINGS_FLAT
```

Comment:
Nit: Macros are mostly placed at the beginning of the line in the code base, even if the code they guard is indented. This is done in a few other places in this PR as well.